packages/devtools_app/test/debugger_model_test.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/debugger/debugger_model.dart';
+import 'package:test/test.dart';
+import 'package:vm_service/vm_service.dart';
+
+void main() {
+  group('debugger_model', () {
+    group('FileNode', () {
+      final ScriptRef ref1 = ScriptRef(uri: 'package:foo', id: 'id-1');
+      final ScriptRef ref2 = ScriptRef(uri: 'package:foo/foo.dart', id: 'id-2');
+      final ScriptRef ref3 = ScriptRef(uri: 'package:bar/bar.dart', id: 'id-3');
+      final ScriptRef ref4 = ScriptRef(uri: 'package:bar/baz.dart', id: 'id-4');
```

Comment:
we should add a test for google3 style uris

Reply:
👍 - added